### ISSUE TITLE
Allow running a single bench from a file

### ISSUE THREAD

**Issue by arjan** [alco/benchfella#58]
Similar to `mix test file:line`, e.g.:

`mix bench bench/template_bench.exs:37`

**Comment by alco** [comment]
Would you like to send a PR implementing this feature? :)

**Comment by arjan** [comment]
Sure, need to find some time for it though :)

**Comment by jdemaris** [comment]
Doesn't this already work? When I specify a file, it runs exactly that one for me with the current version:

```
mix bench bench/graph.exs
```

It only runs that single bench file and not others. That is handled in the paths param that is processed here: https://github.com/alco/benchfella/blob/master/lib/mix/tasks/bench.ex#L100

**Comment by arjan** [comment]
Yes but this is about running just *one* benchmark from a particular file. (you can have multiple benchmarks in a single file).

So adding a colon and the line number to the path argument.

**Comment by arjan** [comment]
I created a PR for this a while back, see #59.

**Comment by jdemaris** [comment]
Oh!!! Nice!! Yea, I saw the PR but I overlooked the details in the original comment. 👍 

Any idea if this is still being maintained? I am starting to use it regularly.

**Comment by arjan** [comment]
I'm not sure... @alco ?

https://github.com/PragTob/benchee seems to be getting more love..

**Comment by jdemaris** [comment]
Yea unfortunately Benchee takes a different approach and everything is centered around comparing different ways of doing the same thing. It really did not fit my benchmarking needs.

I may just have to fork benchfella so we can maintain this. Benchfella is great for exactly what we needed
